bumpfontversion/ufohandler.py

```diff
@@ -19,8 +19,28 @@ def current_version(self, file):
         )
 
     def set_version(self, file, new_version):
+        def pad_num(num, amount):
+            return str(num).zfill(amount)
+
         font = ufoLib2.objects.Font.open(file)
-        font.info.versionMajor = int(new_version["major"].value)
-        font.info.versionMinor = int(new_version["minor"].value)
+        info = font.info
+
+        current_version = f"{info.versionMajor}.{pad_num(info.versionMinor, 3)}"
+
+        info.versionMajor = int(new_version["major"].value)
+        info.versionMinor = int(new_version["minor"].value)
+
+        new_version = f"{info.versionMajor}.{pad_num(info.versionMinor, 3)}"
+
+        # Set OpenType names
+        for attr in dir(info):
```

Comment:
It may be cleaner to just list out the OpenType name records to inspect. However, their names are outrageously long.

Reply:
Don't we just want to change openTypeNameVersion?

Reply:
font version can also appear in the uniqueID record.